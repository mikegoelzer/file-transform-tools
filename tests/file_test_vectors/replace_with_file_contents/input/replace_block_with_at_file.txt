
#
# nvm
#
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
. "$HOME/.cargo/env"

#
# Lattice Diamond license
#
LATTICE_LICENSE_FILE=/usr/local/diamond/3.13/license/license.dat

#
# Added by /home/mwg/ecp5-first-steps/my-designs/util/update_bashrc.sh from 'github.com/mikegoelzer/ecp5-first-steps'
#
export PATH=$PATH:/home/mwg/ecp5-first-steps/my-designs/util/build_helpers/template_tool:/home/mwg/ecp5-first-steps/my-designs/util/continuous_make:/home/mwg/ecp5-first-steps/my-designs/util/slang_tb_gtkwave_helper:/home/mwg/ecp5-first-steps/my-designs/util/clog2
