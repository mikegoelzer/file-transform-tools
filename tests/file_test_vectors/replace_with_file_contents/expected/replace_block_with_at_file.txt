
#
# nvm
#
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
. "$HOME/.cargo/env"

#
# Lattice Diamond license
#
LATTICE_LICENSE_FILE=/usr/local/diamond/3.13/license/license.dat

Hello, world!
It is good to see you.
